### PR TITLE
[PRTL-2694]Add Custom Bins indicator to the table.

### PR DESCRIPTION
**Pull request by yuanhengzhao**
## Ticket Number

e.g. PRTL-2694

## Environment to be used in testing

- [ ] `prod`
- [x] `dev-oicr`
- [ ] `qa` (which version?)

### CODE REVIEW

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js [hunk 1]
```diff
@@ -1282,7 +1282,7 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
 
                     <DropdownItem
                       onClick={() => {
-                        if (resetBinsDisabled) return;
+                        if (!isCustomized) return;
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: could this not be a positive that wraps everything else?

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js [hunk 2]
```diff
@@ -1329,7 +1329,7 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
                       }}
                       style={{
                         ...styles.actionMenuItem,
-                        ...(resetBinsDisabled ? styles.actionMenuItemDisabled(theme) : {}),
+                        ...(isCustomized ? {} : styles.actionMenuItemDisabled(theme)),
```
Comment (caravinci) [inline src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: this needn't be checked, based on the condition in line 1285

File: src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js (hunk 1 above)
Comment (samrichca) [reply src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: did this on the card refactor

```if (binsAreCustom) {
          dispatchUpdateClinicalVariable({
....etc
```

Comment (caravinci) [reply src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard/index.js]: Meaning you're already taking care of this in a later commit? @samrichca

**Pull request merged**